herddb-core/src/test/java/herddb/core/SystemTablesTest.java

```diff
@@ -426,17 +426,17 @@ public void testSystemTablesForNonNullColumnTypes() throws Exception {
         }
     }
 
-    @Test(expected = StatementExecutionException.class)
```

Comment:
Can you please drop these lines instead of commenting them?

Reply:
Yes. Will do.